data/society/schools-libraries/index.html

```diff
@@ -11,17 +11,12 @@
   - schools
 categories: []
 date: 2012-06-13 14:52:45 -0600
-Schools:
-  downloads:
-    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mN1Bobk5uYmNodW8&export=download
-      label: 'Society.Schools: File Geodatabase'
-    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mOXB2NWpKaEJkMWc&export=download
-      label: 'Society.Schools: Shapefile'
```

Comment:
**thought**: Has this been deprecated already?

Reply:
that's the next step.   but, yeah, that layer will be deprecated.

Reply:
ok, in the future we'd want to save this for when that issue hits porter but if you are confident that it is being deprecated then I am ok with it.

Reply:
we'll actually add another layer to this page for higher education schools.  i hope to get that going this week.